src/main/java/com/facebook/openwifirrm/modules/ModelerUtils.java

```diff
@@ -202,4 +209,89 @@ public static double calculateTPCMetrics(
 			return Double.POSITIVE_INFINITY;
 		}
 	}
+
+	/**
+	 * Determines if two wifiscan entries should be aggregated without consideration
+	 * for any kind of obsoletion period. (Untested) should handle APs that support
+	 * pre-802.11n standards (no ht_oper and no vht_oper), 802.11n (supports ht_oper
+	 * but no vht_oper), and 802.11ac (supports both ht_oper and vht_oper).
+	 * Pre-802.11n, if two entries with the same bssid are in the same channel (and
+	 * therefore frequency, since they share a one-to-one mapping), that is enough
+	 * to aggregate them. However, 802.11n introduced channel bonding, so for
+	 * 802.11n onwards, channel width must be checked.
+	 *
+	 * @param entry1
+	 * @param entry2
+	 * @return true if the entries should be aggregated
+	 */
+	private static boolean matchesForAggregation(WifiScanEntry entry1, WifiScanEntry entry2) {
+		// TODO should we check the entire ht_oper and vht_oper or just channel width?
```

Comment:
I'd love feedback from the wireless systems experts here :)

Reply:
Haleema said the basic MCS set does not need to be compared (it represents the intersection of the MCSs supported by all STAs in the network, which is not something that should affect whether we aggregate or not).

Reply:
Yes, this is what I think makes most sense